tests/test_emr.py

```diff
@@ -98,6 +98,28 @@ def test_emr_mark(self):
         tag_map = {t['Key']: t['Value'] for t in new_tags}
         self.assertTrue('test_tag' in tag_map)
 
+    def test_emr_tag(self):
+        session_factory = self.replay_flight_data('test_emr_tag')
+        p = self.load_policy({
+                'name': 'emr-tag-table',
+                'resource': 'emr',
+                'filters': [{'tag:first_tag': 'first'}],
+                'actions': [{
+                    'type': 'tag',
+                    'tags': {'new_tag_key': 'new_tag_value'}
+                }]
+            },
+            session_factory=session_factory)
+        resources = p.run()
+        new_tags = resources[0]['Tags']
```

Comment:
we don't modify refetch/resources with actions, so this wouldn't work on a fresh record, without another policy run.